library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt

```diff
@@ -56,10 +57,13 @@ internal object FormatUtils {
 
     fun formatJson(json: String): String {
         return try {
-            val je = JsonParser.parseString(json)
-            JsonConverter.instance.toJson(je)
-        } catch (e: JsonParseException) {
-            json
+            JSONObject(json).toString(2)
```

Comment:
Let's avoid `magic constants`.
For any other developer the first question here `What is 2 ?`. 
Provide some meaningful name here.

Reply:
Fixed.